pandas/tseries/timedeltas.py

```diff
@@ -62,8 +62,10 @@ def _convert_listlike(arg, box, unit, name=None):
         return Series(values, index=arg.index, name=arg.name, dtype='m8[ns]')
     elif isinstance(arg, ABCIndexClass):
         return _convert_listlike(arg, box=box, unit=unit, name=arg.name)
-    elif is_list_like(arg):
+    elif is_list_like(arg) and getattr(arg, 'ndim', 1) == 1:
```

Comment:
this extra check for `ndim` is needed because a `is_list_like(DataFrame)` is True